README.md

```diff
@@ -1,48 +1,44 @@
 [![XO code
 style](https://img.shields.io/badge/code_style-XO-5ed9c7.svg)](https://github.com/sindresorhus/xo)
-![OpenFaaS](https://img.shields.io/badge/openfaas-serverless-blue.svg)
+![OpenFaaS](https://img.shields.io/badge/openfaas-v0.6.5-blue.svg)
 
 ##### Usage
 
-Add `openfaas` via `npm`
+```js
+const OpenFaaS = require('openfaas')
 
-```
-$ npm install openfaas --save
-```
-
-Example usage
-
-```
-const OpenFaaS = require('./openfaas')
-
-const openfaas = OpenFaaS('http://localhost:8080')
+const faas = new OpenFaaS('http://localhost:8080')
 
-openfaas.deploy(
-	'yolo', // name your function
-	'func_functions, // choose your network
-	'hello-serverless // choose the Docker image
-)
+faas
```

Comment:
This might be because you began with a fork from the original repo, but I think Alex would like to move to `openfaas` over `faas`